michael_tse/test/greet_test.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+var expect = require('chai').expect;
+var greet = require(__dirname + '/../greet');
+
+describe('the greet function', function() {
+  it('should greet someone by name', function() {
+    expect(greet('mike')).to.eql('hello mike');
+  });
+
+});
```

Comment:
Looks good. Might delete some empty lines, and add and empty line at the end of the file in the future. Did you remove the node_modules folder before commit? Instead just add a .gitignore to suppress it from git.